tcms/search/advanced_search.py

```diff
@@ -65,6 +52,26 @@ def advance_search(request):
     return _render_results(request, results, timecost, queries)
 
 
+def _render_errors(request, data, forms):
+    errors = []
+    for form in forms:
+        if form.is_valid():
+            errors.append(form.errors)
+
+    if errors or not data:
+        product_choice = []
+        for product in Product.objects.all():
+            product_choice.append((product.pk, product.name))
+        plan_type_choices = PlanType.objects.all()  # pylint: disable=possibly-unused-variable
+        errors = _fmt_errors(errors)
+        priorities = Priority.objects.filter(  # pylint: disable=possibly-unused-variable
+            is_active=True).order_by('value')
+
+        return render(request, 'search/advanced_search.html', locals())
+
+    return None
```

Comment:
This is redundant and pylint was supposed to flag it. Not sure why it didn't.
FYI when we don't go into the if statement then the default return value of the function is None.